src/eduid_userdb/userdb.py

```diff
@@ -288,3 +288,19 @@ def update_user(self, obj_id, attributes):
                         '$set': attributes,
                     }
                 )
+
+    def get_identity_proofing(self, user):
+        """
```

Comment:
Right. What happened is that the method is in dashboard.userdb.UserDBWrapper. It was needed in user and I copied it to DashboardUserDB. And then I realized it was needed before there was a representation of the user iin the dashboard db, and I moved it over to UserDB.